packages/search/src/elasticsearch/utils.ts

```diff
@@ -403,7 +403,7 @@ export function isValidOperationHistory(body: IBirthCompositionBody) {
   const validStatusMapping = {
     [ARCHIVED_STATUS]: [DECLARED_STATUS, REJECTED_STATUS, VALIDATED_STATUS],
     [IN_PROGRESS_STATUS]: [null],
-    [DECLARED_STATUS]: [ARCHIVED_STATUS],
+    [DECLARED_STATUS]: [ARCHIVED_STATUS, IN_PROGRESS_STATUS, null],
```

Comment:
```suggestion
    [DECLARED_STATUS]: [ARCHIVED_STATUS, null],
```
I don't think we can go from IN_PROGRESS_STATUS to DECLARED_STATUS or can we?